packages/ai/src/workflows/Workflow.ts

```diff
@@ -121,10 +123,15 @@ export class Workflow {
 
   addStep<S extends StepShape>(
     template: LLMStepTemplate<S>,
-    inputs: Inputs<S>
+    inputs: Inputs<S>,
+    retryingStep?: LLMStepInstance<S>
```

Comment:
(minor, easy to refactor later)

`retryingStep` as a third parameter looks arbitrary, we'll eventually get other parameters too. So I'd update to one of:
- `(template, inputs, options?: { retryingStep })`
- `({ template, inputs, retryingStep })`
- `(template, { inputs, retryingStep })`.

The first version is probably the best, because if you agree with my other suggestions then retrying version will be abstracted in `step.retry()`, and the controller code will be clean in the most common two-parameter case.

Reply:
Sure, will do the first now.